meilisearch-core/src/database.rs

```diff
@@ -135,20 +135,34 @@ fn update_awaiter(
     Ok(())
 }
 
+pub struct DatabaseOptions {
+    pub main_map_size: usize,
+    pub update_map_size: usize
+}
```

Comment:
Would it be possible to implement structopt on this struct, declare the `default_values` and `env` variable values here, then add one field to the MeiliSearch `Opt` struct with [the flatten serde atribute](https://serde.rs/field-attrs.html#flatten)?

Reply:
That is possible. But that would mean `StructOpt` would leak into meilisearch-core. Currently all command-line arguments and environment variables are neatly written in `option.rs` of `meilisearch-http`.

Reply:
Yeah, you are probably right, that's better this way, thank you for pointing this!